adobe-esign-connector/webContent/layouts/includes/exception-details.xhtml

```diff
@@ -0,0 +1,109 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html"
+  xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:p="http://primefaces.org/ui"
+  xmlns:pe="http://primefaces.org/ui/extensions"
+  xmlns:c="http://xmlns.jcp.org/jsp/jstl/core">
+
+  <ui:fragment rendered="#{!errorPage.showDetails}">
+    <h:outputText value="An unexpected error has occured:"/>
+  </ui:fragment>
+
+  <h1 class="bg-warning">
+    <h:outputText value="#{errorPage.message}" />
+  </h1>
+
+  <ui:fragment rendered="#{errorPage.isIvyException()}">
+    <h4 class="list-group-item-heading" style="margin-top: 20px">Error id</h4>
+    <p>#{errorPage.exceptionId}</p>
+    <h4 class="list-group-item-heading" style="margin-top: 20px">Error Timestamp</h4>
+    <p>#{errorPage.createdAt}</p>
+  </ui:fragment>
+
+  <ui:fragment rendered="#{errorPage.showDetails}">
+    <h3 class="page-header">
+      <h:outputText value="#{errorPage.error.errorCode}" />
+      <span style="font-size: 0.8em"> 
+        <h:outputText rendered="#{errorPage.error.message!=errorPage.error.errorCode}" value=" #{errorPage.error.message}"/>
+      </span>
+    </h3>
+    <ui:fragment rendered="#{!errorPage.error.attributeNames.isEmpty()}">
+      <h4 class="list-group-item-heading">Attributes</h4>
+      <div class="table-responsive">
+        <table class="table table-bordered table-striped" style="font-size: 1.0em">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <ui:repeat value="#{errorPage.error.attributeNames}" var="attributeName">
+              <tr>
+                <td><h:outputText value="#{attributeName}" /></td>
+                <td style="width: 100%"><h:outputText value="#{errorPage.error.getAttribute(attributeName)}" /></td>
+              </tr>
+            </ui:repeat>
+          </tbody>
+        </table>
+      </div>
+    </ui:fragment>
+    <h4 class="list-group-item-heading">Thrown by</h4>
+    <p style="margin-bottom: 0;">Process:
```

Comment:
Can we use the primeflex styling class of "mb-0" instead of this inline? We have some strict rules for inline CSS with go-live products. Please check the other xhtml file in this PR also. Thank you :)

Reply:
Do you have some proper version of the files without inline styles? Because these template files are generated like this by Ivy and we didn't do any changes to it, thanks 😄

Reply:
I have just tested it again and found that these attributes are generated by default even when we try to override it. So,we can skip these inline CSS as well as the default hint in [frame-10-full-width.xhtml] also